app/assets/stylesheets/earthworks.css

```diff
@@ -134,26 +134,48 @@
   outline: none;
 }
 
+.search-q .input-group {
+  padding: .5rem
+}
+
 .input-group > .search-autocomplete-wrapper {
   border: none;
 }
 
-.search-query-form, .navbar-search .search-query-form{
+.input-group > .search-autocomplete-wrapper input {
+  padding-top: 0;
+  padding-bottom: 0
+}
+
+.search-query-form, .navbar-search .search-query-form {
   border: 1px solid #ABABA9;
-  padding: .5rem;
+  padding: 8px;
   border-radius: .25rem;
   background-color: white;
+  max-height: 47px;
```

Comment:
It shouldn't be necessary to set this.  The height of this is governed by the height of the button and the height of `.search-btn` and `.search-q` If that is constrainted to 29, this is what you will get.

Reply:
The ticket said the button should be 31. But I can let 2 px go!